electrum_nmc/electrum/gui/qt/main_window.py

```diff
@@ -3338,10 +3338,12 @@ def create_buy_names_tab(self):
         self.buy_names_ui = Ui_BuyNamesPage()
         self.buy_names_ui.setupUi(w)
 
-        # TODO: allow hex names
-        self.buy_names_new_name_lineedit = self.buy_names_ui.registerName
+        self.buy_names_new_name_lineedit = self.buy_names_ui.registerNameAscii
```

Comment:
Renaming this from `self.buy_names_new_name_lineedit` to `self.buy_names_new_name_ascii_lineedit` would probably make the code easier to understand.